proposals/NNNN-Avoid-Deadlock.md

```diff
@@ -0,0 +1,31 @@
+# Add a mechanism to avoid Deadlock
+
+* Proposal: [SDL-NNNN](NNNN-Avoid-Deadlock.md)
+* Author: [Yuki Shoda](https://github.com/Yuki-Shoda)
+* Status: Awaiting review
+* Impacted Platforms: [iOS]
+
+## Introduction
+This proposal is to avoid deadlock by adding a mechanism that stops AudioStreaming under certain conditions when the app on HS side moves to background.
```

Comment:
```suggestion
This proposal is to avoid deadlock by adding a mechanism that stops Audio Streaming under certain conditions when the app on HS side moves to background.
```
Also, can you please specify what "HS" refers to here?

Reply:
Can you please change the abbreviation "HS" to the full term?  You can then include "(HS)" after, so that you can use "HS" throughout the rest of the proposal.

Reply:
I've changed the "HS" to "Handset (HS)" in the introduction.